### DISCUSSION TITLE
个人降级的经历

### DISCUSSION THREAD

**Discussion by UsersDisabled** [rA9stuff/LeetDown#110] [Ideas]
个人经验， 2020 M1 mac pro 中
1. 使用命令方式去打开 leetdown 
    cd /Applications/LeetDown.app/Contents/MacOS
    ./leetdown

2.提前下载 firmwares.json 文件 存放到 /tmp 目录下
3. 再提示 5S 内重新插拔设备的时候， 一定要插拔一下
4. 只要 没提示 恢复失败， 就别着急 ，多等会。
5. 特别感谢 各位大佬们的付出。

**Comment by Bilalkhanzaada** [comment]
In english

**Comment by UsersDisabled** [comment]
Personal experience in 2020 M1 mac pro
Use command mode to open leetdown

cd /Applications/LeetDown. app/Content/MacOS
./leetdown

2. Download the firmwares.json file in advance and store it in the /tmp directory
3. When prompted to plug and unplug the device again within 5S, be sure to plug and unplug it
4. As long as there is no prompt for recovery failure, don't worry and wait a little longer.
5. Special thanks to all the big shots for their efforts.
